resources/js/components/user/EditModal.vue

```diff
@@ -197,11 +197,54 @@ const selectFormCourseOptions = useSelectFormCourseOptions(courses, true);
 const selectFormRoleOptions = useSelectFormRoleOptions(roles);
 const randomPlaceholderPerson = usePlaceholderPerson();
 
+const { uploadFileByPresignedUrl } = useS3();
+
 const close = () => {
   emits("close");
 };
 const editSubmitHandler = async () => {
-  Inertia.post(`/dashboard/admin/user/${user.id}`, editForm.value);
+  const avatarPath = ref<string | undefinded>(null);
```

Comment:
You define the type as `string | undefinded` and then set it to `null`. This should then be `ref();` instead of `ref(null);`.